src/Farmer/Types.fs

```diff
@@ -199,9 +199,9 @@ type LinkedResource =
         | Unmanaged resId -> resId
     member this.Name = this.ResourceId.Name
 
-type CertConfig =
-    | AppServiceCertificate of string
-    | NoCertificate of string
+type DomainConfig =
+    | AppServiceDomain of string
+    | NoDomain of string
```

Comment:
I don't think the string argument to `NoDomain` is ever used. Could it be removed?
```fsharp
type DomainConfig =
    | AppServiceDomain of string
    | NoDomain 
```